### PR TITLE
add: image and input component

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Added image and avatar component. Please review these components

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -366,7 +366,76 @@ <h4>Card with Text Overlay</h4>
...
+
+        <!-- ------------------------------------------ Input ------------------------------------ -->
+
+        <div class="component-heading">
+          <h2>Input</h2>
+        </div>
+
+        <div class="component-body">
+          <h4>Basic Input</h4>
+          <div class="component-example-container" style="display: block;">
```
Comment (partha8) [inline index.html:408]: You are using inline styles for preview purposes, right?

**Review by partha8 — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (ApurvaSawant11) [reply index.html:408]: Yes, just for preview purposes. But I'll add a class instead.

**Comment by yash9263** [comment]
Looks good to me

**Pull request merged**